util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -155,24 +155,19 @@
   public static CredentialFactory getCredentialFactory(
```

Comment:
For future proofing, could we add a test ?

Reply:
Added.